format/question/selection/examples/valid/invisible.json

```diff
@@ -0,0 +1,20 @@
+{
+  "id": "1",
+  "type": "application/x.selection+json",
+  "content": "Question ?",
+  "text": "Lorem ipsum sit amet.",
+  "mode": "find",
+  "tries": 10,
+  "solutions":  [
+    {
+      "selectionId": "select1",
+      "answers": [
```

Comment:
I don't think an array of answers makes sense here. You should inline `score`, `begin` and `end` in the parent object.